lib/stream_data.ex

```diff
@@ -1956,6 +1963,39 @@ defmodule StreamData do
     end
   end
 
+  # max_runs and max_run_time both not set by user, default to 100 runs.
```

Comment:
I left all my comments in (that were there to help me plan stuff out) because I figured it would make it easier to review. I plan on taking them out in the next commit.